webrtc.html

```diff
@@ -3855,6 +3856,10 @@
               <dd>A unique identifier that allows ICE to correlate candidates
               that appear on multiple
               <code><a>RTCIceTransport</a></code>s.</dd>
+              <dt><dfn><code>component</code></dfn> of type <span class=
+              "idlAttrType"><a>byte</a></span>, readonly, nullable</dt>
+              <dd>The assigned network component id of the candidate
+              (1 for <code>rtp</code>, 2 for <code>rtcp</code>).</dd>
```

Comment:
i'd rather reference ICE 15.1 at the beginning of the attributes. All but sdpMid, sdpMLineIndex and ufrag are defined there